test/config-lookup/custom-name/custom-name.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const { existsSync } = require('fs');
+const { resolve } = require('path');
+const { run } = require('../../utils/test-utils');
+
+describe('custom config file', () => {
+    it('should work', () => {
+        const { stdout, stderr, exitCode } = run(
+            __dirname,
+            ['--config', resolve(__dirname, 'config.webpack.js'), '--output-path', './binary'],
```

Comment:
The config already have `output.path: resolve(__dirname, 'binary')` , why  passing `--output-path ./binary` again ?

Reply:
Just copy/paste, we need refactor tests